### ISSUE TITLE
GraalVM native image support

### ISSUE THREAD

**Issue by maciejwalkowiak** [webjars/webjars-locator-core#99]
When trying to run Spring PetClinic as a GraalVM native image I hit several issues related to WebJars. As a workaround I added runtime hints: https://gist.github.com/maciejwalkowiak/6f7097157aec2f5d1a4929c0f12bbc67

Perhaps these hints (in standard GraalVM JSON format) could be included in `webjars-locator-core` itself? Or should they be added to https://github.com/oracle/graalvm-reachability-metadata project?

**Comment by sdeleuze** [comment]
IMO a duplicate of https://github.com/webjars/webjars-locator-core/issues/96 which will bring native imahe support if implemented.

**Comment by jamesward** [comment]
Yeah, We need to get #96 dealt with and this should all be better.  I'm working on making progress for that but will leave this open until we get it done and can verify it fixes the GraalVM native image issues.
